integrationTests/testNetwork.go

```diff
@@ -160,29 +164,72 @@ func (net *TestNetwork) MintWalletsUint64(value uint64) {
 }
 
 // SendTx submits the provided transaction to the test network; transaction
-// must be already signed.
+// must be already signed. Returns the transaction hash.
 func (net *TestNetwork) SendTx(tx *transaction.Transaction) string {
 	node := net.firstNodeInShardOfAddress(tx.SndAddr)
 	return net.SendTxFromNode(tx, node)
 }
 
 // SignAndSendTx signs then submits the provided transaction to the test
 // network, using the provided signer account; use it to send transactions that
-// have been modified after being created.
+// have been modified after being created. Returns the transaction hash.
 func (net *TestNetwork) SignAndSendTx(signer *TestWalletAccount, tx *transaction.Transaction) string {
 	net.SignTx(signer, tx)
 	return net.SendTx(tx)
 }
 
 // SendTxFromNode submits the provided transaction via the specified node;
-// transaction must be already signed.
+// transaction must be already signed. Returns the transaction hash.
 func (net *TestNetwork) SendTxFromNode(tx *transaction.Transaction, node *TestProcessorNode) string {
 	hash, err := node.SendTransaction(tx)
 	net.handleOrBypassError(err)
 
 	return hash
 }
 
+// DeploySC deploys a payable contract with the bytecode specified by fileName.
+func (net *TestNetwork) DeployPayableSC(owner *TestWalletAccount, fileName string) []byte {
+	return net.DeploySC(owner, fileName, true)
+}
+
+// DeploySC deploys a non-payable contract with the bytecode specified by fileName.
+func (net *TestNetwork) DeployNonpayableSC(owner *TestWalletAccount, fileName string) []byte {
```

Comment:
the comment doesn't match the function's name

Reply:
done